CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Changelog
 
-## Release 2.5.1 (Unreleased)
+## Release 3.0.0 (Unreleased)
+This release drops support for Alfresco 5.x.
```

Comment:
DE and Alfred API also drop support for 6.0 and 6.1. Up to you if you want to follow suit.

Reply:
6.0 and 6.1 already were no longer supported. 6x compiles for 6.2.2 and we give no guarantees for lower versions